Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -917,8 +917,8 @@ void Detector::constructDetectorGeometry()
 
     LOG(DEBUG1) << "mBuildLevel: " << mBuildLevel[j] << FairLogger::endl;
 
-    if (mStaveThickness[j] != 0) {
-      mGeometry[j]->setStaveThick(mStaveThickness[j]);
+    if (mChipThickness[j] != 0) {
+      mGeometry[j]->setChipThick(mChipThickness[j]);
```

Comment:
Discussing this in the PWG1&2 it turned out to leave the V1 class as a reference at least for some time. Moreover, as Yura pointed out, it could be used again in the future to study a sort of "super-upgraded" version of the ITS. I would leave it for a while in the repository if it is not a problem.